packages/manager/modules/pci/src/components/project/flavors-list/translations/Messages_fr_FR.json

```diff
@@ -14,6 +14,11 @@
   "pci_project_flavors_spec_disk": "Stockage :",
   "pci_project_flavors_spec_bandwidth": "Bande passante :",
   "pci_project_flavors_spec_bandwidth_detail": "{{ bandwidth }} Mbps",
+  "pci_project_flavors_spec_gpu": "Addon :",
+  "pci_project_flavors_spec_gpu_detail_single": "{{ number }} card",
```

Comment:
This is not French, what is the context here so we can help please ? :)

Reply:
While selecting the instance model (b2-7, i-30, g2-15 etc), we need the number of GPU cards to be mentioned as 'Addon' for the GPU models alone. When there is only 1 GPU card, the translation `pci_project_flavors_spec_gpu_detail_single` is used. When there are more than 1 card, the translation `pci_project_flavors_spec_gpu_detail_multi` is used. 

Thanks for offering to help @FredericEspiau  :)

Reply:
Maybe
```suggestion
  "pci_project_flavors_spec_gpu_detail_single": "{{ number }} carte",
```

Reply:
The PU was ok with have the number of cards and the card model in 1 field like `2 x Tesla V100`. Thanks for your inputs @FredericEspiau and @marie-j